$NetBSD: patch-configure,v 1.1 2019/11/06 10:11:41 he Exp $

Use /var/db not /var/lib, to adhere to NetBSD hier(7).

--- configure.orig	2018-09-24 07:52:05.000000000 +0000
+++ configure
@@ -8495,7 +8495,7 @@ full_sysconfdir=`eval eval eval eval eva
 full_localstatedir=`eval eval eval eval eval echo "${localstatedir}" | sed "s#NONE#${prefix}#" | sed "s#NONE#${ac_default_prefix}#"`
 full_libdir=`eval eval eval eval eval echo "${libdir}" | sed "s#NONE#${prefix}#" | sed "s#NONE#${ac_default_prefix}#"`
 default_softhsm2_conf="`eval echo ${full_sysconfdir} | sed s,NONE,$ac_default_prefix,g`/softhsm2.conf"
-softhsmtokendir=${full_localstatedir}/lib/softhsm/tokens/
+softhsmtokendir=${full_localstatedir}/db/softhsm/tokens/
 
 # Install the library in a sub-directory
 full_libdir="$full_libdir/softhsm"
